src/deluge/gui/views/arranger_view.cpp

```diff
@@ -935,6 +935,10 @@ void ArrangerView::auditionEnded() {
 }
 
 ActionResult ArrangerView::padAction(int32_t x, int32_t y, int32_t velocity) {
+	// do not interpret pad actions when stem export is underway
```

Comment:
can this be done at the matrix driver instead of spreading these checks everywhere?